rfd/0241-scoped-access-lists.md

```diff
@@ -0,0 +1,931 @@
+---
+authors: Nic Klaassen (nic@goteleport.com)
+state: draft
+---
+
+# RFD 241 - Scoped Access Lists
+
+## Required Approvers
+
+* Engineering: @fspmarshall && (@r0mant || @smallinsky || @kopiczko || @hugoShaka)
+* Security: @rosstimothy || @rob-picard-teleport
+* Product: @klizhentas
+
+## What
+
+At its core, an access list is a list of users and a set of privileges to be
+assigned to all users in the list.
+Scoped access lists are a new scoped variant of traditional access lists
+that can be used to grant scoped roles to groups of users, and to grant those
+roles at specific scopes.
+
+Required Reading:
+
+- [RFD 229 - Scopes](./0229-scopes.md)
+
+Recommended Reading:
+
+- [RFD 6e - Access Lists](https://github.com/gravitational/teleport.e/blob/master/rfd/0006e-access-lists.md)
+- [RFD 170 - Nested Access Lists](./0170-nested-accesslists.md)
+
+Scoped access lists will initially aim to provide a small subset of the
+functionality of traditional access lists, tailored for the specific use case
+of using scoped access lists to manage scoped privileges in the context of
+automation and identity provider sync.
```

Comment:
>  manage scoped privileges in the context of automation and identity provider sync.

If identity provider sync is main use case  RFD is missing main details. 

Current for of scoped access list says that "scoped access lists will likely never support the concept of .. traits" 


So it will mean that in case of integration like EntraID identity sync well never support scoped access list because EntraID group sync leverages the access list traits feature where 90% of the marked is either EntraID or Okta IdP.